components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Registry/src/Deploy/IoTHubDiscovererDeployment.cs

```diff
@@ -95,15 +95,7 @@ public sealed class IoTHubDiscovererDeployment : IHostProcess {
                     Cmd = new[] {
                         "PkiRootPath=/mount/pki"
                     },
-                    NetworkingConfig = new {
-                        EndpointsConfig = new {
-                            host = new {
-                            }
-                        }
-                    },
                     HostConfig = new {
-                        NetworkMode = "host",
```

Comment:
Something that just occurred to me: Will auto scanning still work without the Discovery module being in the host network?

Reply:
Yes, it works fine. With default settings it will go on all the network interfaces available to the discovery module and when a certain IP address range is specified, it will scan that particular range.